master/buildbot/process/buildstep.py

```diff
@@ -731,7 +731,7 @@ def workerVersionIsOlderThan(self, command, minversion):
         sv = self.build.getWorkerCommandVersion(command, None)
         if sv is None:
             return True
-        if map(int, sv.split(".")) < map(int, minversion.split(".")):
+        if list(map(int, sv.split("."))) < list(map(int, minversion.split("."))):
```

Comment:
List comprehension would work fine here.  I don't feel strongly one way or the other.